packages/service/src/domain/commands/processors/tag-remove-command.ts

```diff
@@ -0,0 +1,29 @@
+import Command from '../model/command';
+import CommandProcessingResponse, { makeSingleTextProcessingResponse } from '../model/command-processing-response';
+import CommandDefinition from '../model/command-definition';
+import XSoundsService from '../../x-sounds/x-sounds-service';
+
+async function tagRemoveCommandProcessor(command: Command): Promise<CommandProcessingResponse> {
+  const commandArgs = command.getArgsByDelimiter('|');
+
+  if (commandArgs.length < 2) {
+    throw new Error('Zbyt mała liczba argumentów');
```

Comment:
Maybe lets provide correct usage in response instead of returning generic message

Reply:
I thought about it, but this seems like feature that all commands could use. This is beyond the scope of this PR in my opinion.

Reply:
We could adres it locally for tag-* in this PR and "globally" as separate issue. But I will not insist.

Reply:
You're right - I'll change that